core/topology/topologyHandler_test.go

```diff
@@ -0,0 +1,129 @@
+package topology
+
+import (
+    "testing"
+    "time"
+
+    "github.com/ElrondNetwork/elrond-eth-bridge/testsCommon"
+    "github.com/stretchr/testify/assert"
+)
+
+var duration = time.Second
+
+func TestNewTopologyHandler(t *testing.T) {
+    t.Parallel()
+
+    t.Run("should work", func(t *testing.T) {
+        providedSortedPublicKeys := [][]byte{[]byte("aaa"), []byte("bbb")}
+        providedTimer := createTimerStubWithUnixValue(123)
+        providedAddress := []byte("aaa")
+        providedStepDuration := duration
+        args := ArgsTopologyHandler{
```

Comment:
create a function createStubTopologyHandler that constructs the ArgsTopologyHandler and call  NewTopologyHandler returning the instance. This can be reused in all the tests reducing the number of lines.